users/hasher/hasher.go

```diff
@@ -13,7 +13,7 @@ const cost int = 10
 
 var (
 	errHashPassword    = errors.New("Generate hash from password failed")
-	errComparePassword = errors.New("Compare hash and password failed")
+	errComparePassword = errors.New("Failed to verify identity or secret")
```

Comment:
Make these errors lowercase. Also, this error is specific to the actual secret so it does not make sense to mention identity here.